src/ripple/overlay/impl/ProtocolMessage.h

```diff
@@ -25,14 +25,17 @@
 #include <ripple/overlay/Message.h>
 #include <ripple/overlay/impl/ZeroCopyStream.h>
 #include <ripple/protocol/messages.h>
+#include <ripple.pb.h>
+
 #include <boost/asio/buffer.hpp>
 #include <boost/asio/buffers_iterator.hpp>
 #include <boost/system/error_code.hpp>
+#include <google/protobuf/message.h>
```

Comment:
It is "unneeded" only because it is transitively included elsewhere, but its exports are directly used on lines 286 and 321, so I prefer to leave it in.